static/js/chart/draw_scatter.ts

```diff
@@ -536,18 +550,37 @@ function addRegressionLine(
   );
 }
 
+/**
+ * Options that can be set for how the scatter plot is drawn.
+ */
+interface ScatterPlotOptions {
+  xPerCapita: boolean;
+  yPerCapita: boolean;
+  xLog: boolean;
+  yLog: boolean;
+  showQuadrants: boolean;
+  showDensity: boolean;
+  showLabels: boolean;
+  showRegression: boolean;
+}
+
 /**
  * Plots a scatter plot.
  * @param svg
  * @param tooltip
  * @param props
  */
 export function drawScatter(
-  svgContainerRef: React.MutableRefObject<HTMLDivElement>,
-  tooltipRef: React.MutableRefObject<HTMLDivElement>,
+  svgContainerRef: React.RefObject<HTMLDivElement>,
+  tooltipRef: React.RefObject<HTMLDivElement>,
   chartWidth: number,
   chartHeight: number,
-  props: ChartPropsType,
+  points: { [placeDcid: string]: Point },
+  yLabel: string,
```

Comment:
Maybe have a struct to hold "chartWidth", "chartHeight", "x/y Label", "x/y Unit"? In general, we prob should reduce the number of argument of functions in general..

Reply:
done